control/tests/freqresp_test.py

```diff
@@ -508,3 +509,17 @@ def test_dcgain_consistency():
 
     sys_ss = ctrl.tf2ss(sys_tf)
     np.testing.assert_almost_equal(sys_ss.dcgain(), -1)
+
+
+def test_singular_values_plot():
+    den = [75, 1]
+    sys = tf([[[87.8], [-86.4]],
+              [[108.2], [-109.6]]],
+             [[den, den],
+              [den, den]])
+    sigma, omega = singular_values_plot(sys, 0.0)
+    sys_dc = np.array([[87.8, -86.4],
+                       [108.2, -109.6]])
+
+    u, s, v = np.linalg.svd(sys_dc)
```

Comment:
Use static reference values

Reply:
What do you mean? 
First I was thinking to use the reference mimo system from function ss_mimo(), but it is actually MISO (2 in, 1 out)

Reply:
```
s = np.array([197.20868123,   1.39141948])
```

parametrize together with the system (see e.g. timeresp_test.py)